modules/migration-compatibility-guidelines.adoc

```diff
@@ -10,16 +10,43 @@
 
 You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
 
-You cannot install {mtc-short} {mtc-version} on {product-title} 4.5, or earlier versions, because the custom resource definition API versions are incompatible.
+.Definitions
 
-You can migrate workloads from a source cluster with {mtc-short} {mtc-legacy-version-z} to a target cluster with {mtc-short} {mtc-version} as long as the `MigrationController` custom resource and the {mtc-short} web console are running on the target cluster.
+legacy platform:: {product-title} 4.5 and earlier
```

Comment:
Is there a reason these are all lowercased? Some kind of guideline?

Reply:
@eriknelson 
Just convention. For example, see the glossary of terms here: https://redhat-documentation.github.io/supplementary-style-guide/#_a

These words are not proper nouns, so lower case seems more appropriate.